README.md

```diff
@@ -195,9 +196,9 @@ If you don't want to depend on Docker Compose but only Docker, you can launch se
 It may take a long time to build dependencies when launching servers first time.
 
 ```sh
-$ docker build --target dev-api -t photoview-api . # Build image for development
-$ cp api/example.env api/.env
-$ docker run --rm -it -v `pwd`:/app --network host photoview-api # Monitor source code and (re)launch API server
+$ docker build --target api -t photoview/api . # Build image for development
+$ docker run --rm -it -v `pwd`:/app --network host --env-file api/example.env photoview/api \
```

Comment:
Why do you use `--network host` here? It is fine in the scope of the CI, as the runner is anyway isolated. But here for the developer environment, this might cause some issues and introduce unnecessary security issues. Is there a way to avoid this?

Reply:
Yes, but it makes the dev environment more complex to set. It requires to [create a new Docker network](https://docs.docker.com/network/) and launch two servers in that network. With that network, the address changes from `localhost` to some random address depending on how a developer configs Docker.

As a development env is not a permanent environment, I prefer convenient over security.

Reply:
Oh, I forgot to say, UI server needs to access API server, so they need to be in the same network, or `--link`.

Reply:
Exposing ports is not a drawback. As it's to set local dev env, exposing ports locally is easy to debug and check if a service runs well.

I'll try to limit the server to listening `localhost` only. It avoids most of security issues.

Besides the network part, do you have any other things to document?

Reply:
> Besides the network part, do you have any other things to document?

Except of already mentioned topics I don't see now something else missing. Anyway, documentation-related changes can be done at any time, as they don't have any dependencies

Reply:
Done.

Reply:
BTW, `localhost` doesn't work well with `docker compose`, as it can't expose ports on localhost. It needs to listen the address assigned by the network.